src/backend/users/views.py

```diff
@@ -98,19 +99,21 @@ class UserProfileView(APIView):
 
     def repackage_for_frontend(self, serialized_data):
         # repackage destinations to match frontend AccountAddress
-        # TODO replace dummy Point data issue 486
-        # (https://github.com/azavea/echo-locator/issues/486)
         user_profile = serialized_data["userprofile"]
+        print(user_profile["destinations"])
```

Comment:
Did you mean to leave this in here?

Reply:
Oops, sure didn't! Thanks for catching that!